shared-data/js/helpers/getLabwareInfoByLiquidId.ts

```diff
@@ -2,13 +2,97 @@ import reduce from 'lodash/reduce'
 
 import type { LoadLiquidRunTimeCommand, RunTimeCommand } from '../../command'
 
+export const consolidateSharedWells = (
```

Comment:
```suggestion
const consolidateSharedWells = (
```
Let's move this function beneath the major exported util, too.